Javastructure.js

```diff
@@ -0,0 +1,98 @@
+const cardData = [
```

Comment:
I don't think you should name the file `Javasctructure.js`. `Java` is a completely different language than `JavaScript`. How about `renderCards.js`? To describe, what it actually does?